ext/nnstreamer/include/nnstreamer.fbs

```diff
@@ -0,0 +1,36 @@
+namespace NNStreamer;
```

Comment:
Please explain the purpose of this file.
For now, let's place this along with other flabuf c/h files and create RPM/DEB subpackages for flatbuf developers. We may need to see how other ubuntu/debian packages do in the same situation.